test/node-api/test_buffer/test_buffer.c

```diff
@@ -7,15 +7,21 @@ static const char theText[] =
     "Lorem ipsum dolor sit amet, consectetur adipiscing elit.";
 
 static int deleterCallCount = 0;
-static void deleteTheText(napi_env env, void* data, void* finalize_hint) {
+
+static void deleteTheText(node_api_basic_env env,
+                          void* data,
+                          void* finalize_hint) {
   NODE_API_ASSERT_RETURN_VOID(
```

Comment:
Please use `NODE_API_BASIC_ASSERT_RETURN_VOID` here instead of performing type coercion in `NODE_API_ASSERT_BASE` as it is not valid to throw JS exception in basic finalizers.

Reply:
thank you